### PR TITLE
feat:-  added typography, lists, navigation, and modal

### PR DESCRIPTION
**Pull request by 05him**
added typography, lists, navigation, and modal components in the component library

live URL:- https://05him.github.io/Dark_UI/

### CODE REVIEW

**Review by aryanandarnav13 — APPROVED** [review]
Looks Good

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.modal-box {
+  background: var(--text-color);
+  color: var(--card-text);
```
Comment (sunilvr) [inline components/modal/modal.css:3]: color variables can have consistent naming conventions

**Comment by sunilvr** [comment]
good to be merged Himanshu.

**Review by sunilvr — APPROVED** [review]

File: components/modal/modal.css (hunk 1 above)
Comment (05him) [reply components/modal/modal.css:3]: should i  change it to --card-text-color?

**Pull request merged**